src/plone/volto/profiles.zcml

```diff
@@ -24,42 +24,6 @@
       post_handler=".setuphandlers.uninstall"
       />
 
-  <genericsetup:registerProfile
-      name="default-homepage"
-      title="Plone 6 Frontend (Default content on homepage)"
-      description="Creates a default page for the site"
-      provides="Products.GenericSetup.interfaces.EXTENSION"
-      directory="profiles/homepage"
-      post_handler=".setuphandlers.create_default_homepage"
-      />
-
-  <genericsetup:registerProfile
-      name="default-homepage-drafjs"
-      title="Plone 6 Frontend (Default content on homepage with draftJS blocks)"
-      description="Creates a default page for the site using (legacy) draftJS blocks"
-      provides="Products.GenericSetup.interfaces.EXTENSION"
-      directory="profiles/homepage"
-      post_handler=".setuphandlers.create_default_homepage_draftjs"
-      />
-
-  <genericsetup:registerProfile
-      name="default-homepage-slate"
-      title="Plone 6 Frontend (Default content on homepage with slate blocks)"
-      description="Creates a default page for the site using slate blocks"
-      provides="Products.GenericSetup.interfaces.EXTENSION"
-      directory="profiles/homepage"
-      post_handler=".setuphandlers.create_default_homepage_slate"
-      />
-
-  <genericsetup:registerProfile
-      name="demo"
-      title="plone.volto demo special requirements"
-      description="Special requirements and setup for demo site"
-      provides="Products.GenericSetup.interfaces.EXTENSION"
-      directory="profiles/demo"
-      post_handler=".setuphandlers.create_demo_homepage"
-      />
```

Comment:
Removing these profiles should be mentioned as a breaking change in the release notes.